node/src/actors/chain_manager/mod.rs

```diff
@@ -1342,6 +1342,22 @@ impl ChainManager {
                     // the list itself is ordered by decreasing reputation
                     let ars_identities = ARSIdentities::new(ars_members);
 
+                    // TODO: decide hard fork date and minimum value
+                    // After the Nth hard fork, the minimum superblock committee size must be 1/3 of 100
+                    let superblock_hard_fork_index = 99999999;
+                    let min_committee_size = if superblock_index > superblock_hard_fork_index {
+                        // Note that this ignores the case when the ARS size is less that 100.
+                        // For example, if the ARS size is 10, then the minimum will be 100/3 = 33,
+                        // so all 10 identities are expected to vote, and 2/3 of that 10 identities
+                        // must be in consensus.
+                        // If we want to ask for 1/3 of 10 in that case, so 3 identities , we need to use
+                        // min(ars_identities.len(), consensus_constants.superblock_signing_committee_size) / 3
+                        consensus_constants.superblock_signing_committee_size / 3
```

Comment:
This leaves me wondering: Is there any evidence of the ARS size ever going that low? Is it even possible in a successfully bootstrapped network?

Reply:
In a normal scenario, it should never go below 2000 as that is the number of epochs a node is considered active when mining a block? The only cases I can think of would be long-ish rollbacks (e.g. longer than 2000 epochs) due to consensus failures causing a large majority of the nodes to fail, but even that sounds somewhat unlikely.